setup.py

```diff
@@ -78,7 +78,7 @@
             "docker",
             "backoff",
             "psycopg2-binary",
-            "pytest-asyncio==0.10.0",
+            "pytest-asyncio",
```

Comment:
So we can pin the version in the application depending on the pytest version (pytest 6.0.0 requires pytest-asyncio > 0.10.0)

Reply:
Yes, it's really not a good idea to force a specific version.